backend/src/main/java/quarano/core/validation/AlphaNumeric.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package quarano.core.validation;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static quarano.core.validation.AlphaNumeric.ERROR_MSG;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import javax.validation.constraints.Pattern;
+
+/**
+ * @author Oliver Drotbohm
+ */
+@Pattern(regexp = Strings.LETTERS, message = ERROR_MSG)
+@Documented
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER, TYPE_USE })
+@Retention(RUNTIME)
+@Constraint(validatedBy = {})
+public @interface AlphaNumeric {
+    String ERROR_MSG = "{Pattern.alphaNumeric}";
+
+    String message() default ERROR_MSG;
```

Comment:
Do we need these fields if we don't make any use of them anywhere for now?

Reply:
Unfortunately, they are needed at @Constraint annotations. If they are not provided, we'll get a RuntimeException on Request for each of these 3 fields.
HV000074: quarano.core.validation.Name contains Constraint annotation, but does not contain a groups parameter.